src/icons/icons/arrowIcon.ts

```diff
@@ -0,0 +1,4 @@
+import { ArrowIcon } from './svgs';
```

Comment:
Arrow is only ever used in Tooltip, I wonder if we should just move the generated component to the tooltip folder and remove all the related Arrow files in icons, I dont think design will have an use for ArrowIcon in their pages wireframes

Reply:
I added this SVG while I was revamping tooltip. I didnt know where to put it to get the SVGR, now that we can generate and commit the generated TSX I think we should move it to tooltip folder, unless design team have some future usage for this

Reply:
It is used in Alert usage in user-land: https://github.com/curology/radiance-ui/pull/386